dotnet-engine/Yggdrasil.Engine.Tests/YggdrasilEngineTest.cs

```diff
@@ -220,4 +220,20 @@ public void Impression_Data_Test_Disabled()
         Assert.IsTrue(result);
         Assert.IsFalse(shouldEmit);
     }
+
+    [Test]
+    public void Invalid_Json_Raises_An_Error()
```

Comment:
This doesn't do anything related to this PR but these tests were missing and they kinda should be here